fsw-react-calculator/src/Components/Buttons.js

```diff
@@ -0,0 +1,143 @@
+import { Component } from "react";
+class Buttons extends Component {
```

Comment:
Because this component doesn't track state, think about updating to be a functional component

Reply:
Successfully converted to functional component